pom.xml

```
@@ -164,14 +154,12 @@
     <dependency>
       <groupId>io.jenkins.lib</groupId>
       <artifactId>support-log-formatter</artifactId>
-      <!-- 1.3 requires Java 17 -->
-      <version>1.2</version>
+      <version>1.4.1</version>
```

I don't see a 1.4.1 release of support log formatter in its [releases](https://github.com/jenkinsci/lib-support-log-formatter/releases).  There is a 1.3 release
```suggestion
      <version>1.3</version>
```

---

Thanks, looks like I screwed up my local Maven repo somehow.